generic/GenericCplComp.F90

```diff
@@ -1208,7 +1209,7 @@ subroutine COUPLE(SRC, STATE, RC)
 
           end select
 
-          STATE%ACCUM_COUNT(J) = -1
+          !STATE%ACCUM_COUNT(J) = -1
```

Comment:
```suggestion
          STATE%ACCUM_COUNT(J) = 0
```

Reply:
@atrayano is setting to zero, really the right behavior there. Shouldn't that be the providence of the "time_to_zero" alarm. All the "time_to_couple" says is take whatever I have and fill the destination arrays if I understand right, why reset anything at all? Shouldn't that be responsibility of the time to zero alarm?

Reply:
My mistake, please ignore the suggested change. I was thinking this is happening during initialization